databrowser/src/pages/HomePage.vue

```diff
@@ -1,120 +1,121 @@
+            <span>Show API Documentation</span></InternalLink
+          >
+        </CardContainer>
 
-      <CardContainer tag-name="li">
-        <CardTitle>Information</CardTitle>
-        <CardText
-          >Lorem ipsum dolr sit amen dol rist maen dol ist amen dorl sit amen
-          dorl sit amen dol dori sti ist.
-        </CardText>
-        <CardDivider />
-        <InternalLink to="/" tone="primary">
-          <ArrowRight class="fill-current" />
-          <span>Show Information</span></InternalLink
-        >
-      </CardContainer>
-    </CardGrid>
+        <CardContainer tag-name="li">
+          <CardTitle>Information</CardTitle>
+          <CardText
+            >Lorem ipsum dolr sit amen dol rist maen dol ist amen dorl sit amen
+            dorl sit amen dol dori sti ist.
+          </CardText>
+          <CardDivider />
+          <InternalLink to="/" tone="primary">
+            <ArrowRight class="fill-current" />
+            <span>Show Information</span></InternalLink
+          >
+        </CardContainer>
+      </CardGrid>
 
-    <ContentDivider />
+      <ContentDivider />
 
-    <CardGrid tag-name="ul">
-      <li>
-        <router-link to="/error">Go to Error</router-link>
-      </li>
-      <li>
-        <router-link to="/dataset/odh-activity-poi"
-          >ODH ActivityPOI List
-        </router-link>
-      </li>
-      <li>
-        <router-link to="/dataset/odh-activity-poi/smgpoi107"
-          >ODH ActivityPOI Detail (ID = smgpoi107)
-        </router-link>
-      </li>
-      <li>
-        <router-link to="/dataset/odh-activity-poi/smgpoi107/raw"
-          >ODH ActivityPOI Raw (ID = smgpoi107)
-        </router-link>
-      </li>
-      <li>
-        <router-link to="/dataset/odh-activity-poi-types"
-          >ODH ActivityPOITypes List
-        </router-link>
-      </li>
-      <li class="py-4">
-        <router-link class="bg-red-100" to="/dataset/xyz"
-          >Not existing list route
-        </router-link>
-      </li>
-    </CardGrid>
-  </ContentArea>
+      <CardGrid tag-name="ul">
+        <li>
+          <router-link to="/dataset/odh-activity-poi"
+            >ODH ActivityPOI List
+          </router-link>
+        </li>
+        <li>
+          <router-link to="/dataset/odh-activity-poi/smgpoi107"
+            >ODH ActivityPOI Detail (ID = smgpoi107)
+          </router-link>
+        </li>
+        <li>
+          <router-link to="/dataset/odh-activity-poi/smgpoi107/raw"
+            >ODH ActivityPOI Raw (ID = smgpoi107)
+          </router-link>
+        </li>
+        <li>
+          <router-link to="/dataset/odh-activity-poi-types"
+            >ODH ActivityPOITypes List
+          </router-link>
+        </li>
+        <li class="py-4">
+          <router-link class="bg-red-100" to="/dataset/xyz"
+            >Not existing list route
+          </router-link>
+        </li>
+      </CardGrid>
+    </ContentArea>
+  </AppLayout>
 </template>
+
 <script>
```

Comment:
I would set the `lang="ts"` info to improve tooling support (e.g. by VSCode `volar` extension).

At the moment, `volar` shows me this message:

```
Virtual script not found, may missing <script lang="ts"> / "allowJs": true / jsconfig.json.volar
```